odd_collector/adapters/fivetran/adapter.py

```diff
@@ -0,0 +1,38 @@
+from odd_models.models import DataEntityList
+from oddrn_generator import FivetranGenerator
+
+from .helpers import DatasetGenerator
+from .mappers.transformers import map_transformers
+from .repository import FivetranRepository
+from odd_collector_sdk.domain.adapter import AbstractAdapter
+from odd_collector.domain.plugin import FivetranPlugin
+
+
+class Adapter(AbstractAdapter):
```

Comment:
Use `BaseAdapter` or `AsyncAbstractAdapter` instead of `AbstractAdapter` (will be deprecated)